valet/helpers_test.go

```diff
@@ -26,15 +26,17 @@ import (
 	"math/rand"
 	"os"
 	"path/filepath"
+	"sort"
 
 	ex "github.com/kjsanger/extendo"
 	. "github.com/onsi/ginkgo"
 
 	"github.com/kjsanger/valet/utilities"
+	"github.com/kjsanger/valet/valet"
 )
 
 type itemPathTransform func(i []ex.RodsItem) []string
-type localPathTransform func(p []string) []string
+type localPathTransform func(p []valet.FilePath) []string
```

Comment:
Why isn't this and other usage using your new FilePathArr type?

Reply:
As it's not interchangeable with `[]valet.FilePath` I'd rather not use it except where I have to, because it would mean an extensive change. I've swapped it for a `valet.SortFilePaths` function because that's simpler and doesn't leave untidy exported symbols.